### PR TITLE
My work day

### PR DESCRIPTION
**Pull request by YehuditShrg**
all the scraper (3...) that use my word day.

### CODE REVIEW

**Review by snirsh — CHANGES_REQUESTED** [review]
we can use JSON and not webdriver, no?

File: ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+class MyWorkDayScraper(Scraper):
+    url = ''
+    name = ''
+
+    def __init__(self):
+        super().__init__()
+        self.driver = self.selenium_url_maker(self.url)
+
+    def scrape(self):
+        driver = Scraper.selenium_url_maker(self.url)
```
Comment (snirsh) [inline ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py:23]: Can we use requests ?

**Review by snirsh — APPROVED** [review]

**Review by snirsh — CHANGES_REQUESTED** [review]
Can we try removing the driver?
Only using requests/soup is better and more stable.

WDYT?

File: ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        }
+        headers["User-Agent"] = generate_user_agent()
+
+        response = requests.post(url, headers=headers)  # cookies=cookies, )#, data=data)
+
+        print(response.status_code)
+        print(response.text)
+
+
+MyWorkDayScraper().scrape()
```
Comment (snirsh) [inline ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py:85]: Remove and remove comments

File: ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
...iptor":"US, NY, New York","id":"d2088e737cbb01293f745dc7ce017070","count":10},{"descriptor":"US, NY, Remote","id":"91336993fab910af6d6ff2800b0cc25c","count":17},{"descriptor":"US, OR, Hillsboro","id":"91336993fab910af6d7027195454c2d4","count":48},{"descriptor":"US, OR, Remote","id":"91336993fab910af6d6ff05591bcc257","count":12},{"descriptor":"US, Remote","id":"16fc4607fc4310011e929f7115f90000","count":59},{"descriptor":"US, SC, Remote","id":"91336993fab910af6d6ffb099324c270","count":1},{"descriptor":"US, TX, Austin","id":"91336993fab910af6d702b631b94c2de","count":113},{"descriptor":"US, TX, Austin-Braker Pointe","id":"970bf8c909a701a82deefadcd4009f07","count":1},{"descriptor":"US, TX, Remote","id":"91336993fab910af6d702939a7fcc2d9","count":38},{"descriptor":"US, VA, Herndon","id":"c498fba66f4e012b5a8a2e88d5007805","count":1},{"descriptor":"US, VA, Remote","id":"91336993fab910af6d7159be65c4c3ed","count":4},{"descriptor":"US, WA, Redmond","id":"91336993fab910af6d701e82d004c2c0","count":33},{"descriptor":"US, WA, Remote","id":"91336993fab910af6d7169a81124c410","count":31},{"descriptor":"US, WA, Seattle","id":"d2088e737cbb01d5e2be9e52ce01926f","count":7}]}]}],"userAuthenticated":false}
```
Comment (snirsh) [inline ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py:88]: What is this? 😅

**Review by snirsh — CHANGES_REQUESTED** [review]

File: ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py (hunk 3 above)
Comment (YehuditShrg) [reply ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py:88]: we try use request . it is in progress.

Comment (YehuditShrg) [reply ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py:88]: we tried using request but it returns a JSON only with the 20 first jobs and for the next job we need to send some meta data like cookies and we face problem with it

Comment (YehuditShrg) [reply ScrapingTools/OutsourceTools/MyWorkDay/MyWorkDayScraper.py:88]: can you take a look how to solve it?

**Pull request closed without merging**